src/main/java/software/aws/neptune/opencypher/OpenCypherIAMRequestGenerator.java

```diff
@@ -62,8 +69,7 @@ public static AuthToken createAuthToken(final String url, final String region) {
         signer.setRegionName(region);
         signer.setServiceName(request.getServiceName());
         signer.sign(request, AWS_CREDENTIALS_PROVIDER.getCredentials());
```

Comment:
Lines 63 to 71 can be removed, they are duplicated and used below, just need to return the token.